### PR TITLE
Tags negative tests

### PR DESCRIPTION
**Pull request by jimytz**

### CODE REVIEW

File: src/main/java/constants/DomainAppEnums.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package constants;
```
Comment (Agustin-Mediotti) [inline src/main/java/constants/DomainAppEnums.java:1]: This Enum should be the same that in [dev](https://github.com/AT15-TestNG/WordPress-API/blob/dev/src/main/java/constants/DomainAppEnums.java) branch. Take a look

File: src/test/resources/features/posts/tags/negative.feature [hunk 2]
```diff
@@ -0,0 +1,38 @@
+@Tags @Negative
```
Comment (Agustin-Mediotti) [inline src/test/resources/features/posts/tags/negative.feature:1]: Add @Regression here

File: src/test/resources/features/posts/tags/negative.feature [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Feature: Tags Negative tests
+
+  @DeleteTagError404 @Regression @Bug
```
Comment (Agustin-Mediotti) [inline src/test/resources/features/posts/tags/negative.feature:4]: @Regression should not be in the Tests, only above.

**Review by Agustin-Mediotti — CHANGES_REQUESTED** [review]
Fix those things and you are ok!

File: src/test/resources/features/posts/tags/negative.feature (hunk 2 above)
Comment (jimytz) [reply src/test/resources/features/posts/tags/negative.feature]: Done!

File: src/test/resources/features/posts/tags/negative.feature (hunk 3 above)
Comment (jimytz) [reply src/test/resources/features/posts/tags/negative.feature]: Got it!

File: src/test/resources/features/posts/tags/bug.feature [hunk 4]
```diff
@@ -0,0 +1,13 @@
+@Tags @Bugs
```
Comment (Agustin-Mediotti) [inline src/test/resources/features/posts/tags/bug.feature:1]: @Bug in singular

**Review by Agustin-Mediotti — DISMISSED** [review]
@Bug tag should be in singular

**Review by eliosfg — APPROVED** [review]
Nice!

**Pull request merged**